### PR TITLE
DOCS-495: Clean up SNSD Pre-Existing Data behavior

### LINKED ISSUE #495: Refine Behavior for SNSD Deployments
**Summary**

There's some confusion around what happens when you start a Single Node Single Drive (SNSD) deployment against existing data. As per engineering:

- Fresh Backend -> Create XL Backend, Start Normally
- Existing SNSD XL Backend -> Start normally in XL Mode
- Existing FS Backend -> Start normally in FS Mode
- New FS Backend -> Fail

The key is that if you were running MinIO against the Filesystem initially, then upgrade, things _should_ still work. Will need to test this to confirm if there are edge cases.

**Goals**

List the in-scope goals

- [ ] Update the [SNSD](https://docs.min.io/minio/baremetal/installation/deploy-minio-single-node-single-drive.html#deploy-minio-as-single-node-single-drive) docs to include a table similar to the above
- [ ] Double check each scenario

**Non-Goals**

Any other rewrites, optimizations, or clean ups

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by ravindk89**
Closes #495 

# Summary

Update the SNSD Pre-Existing Data section to better define the specific outcomes of running SNSD mode against existing (or non-existing) data.

# Non-Goals

- Fixing the deployment comparison matrix
- Updating the other deployment types for pre-existing data behavior
- Any other clean ups

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

File: source/installation/deploy-minio-single-node-single-drive.rst [hunk 1]
```diff
@@ -11,26 +11,42 @@ Deploy MinIO as Single-Node Single-Drive
...
+.. _minio-snsd-pre-existing-data:
 
 Pre-Existing Data
 -----------------
 
-Starting with :minio-release:`RELEASE.2022-06-02T02-11-04Z`, MinIO implements a zero-parity erasure coded backend for single-node single-drive deployments.
-This feature allows access to :ref:`erasure coding dependent features <minio-erasure-coding>` without the requirement of multiple drives.
-The specified data volume or folder *must* be empty *or* only contain files generated by a previous |SNSD| deployment.
+MinIO startup behavior depends on the the contents of the specified storage volume or path.
+The server checks for both MinIO-internal backend data and the structure the structure of existing folders and files.
```
Comment (djwfyi) [inline source/installation/deploy-minio-single-node-single-drive.rst:28]: `the structure` is duplicated

File: source/installation/deploy-minio-single-node-single-drive.rst [hunk 2]
```diff
@@ -11,26 +11,42 @@ Deploy MinIO as Single-Node Single-Drive
...
+       
+     - MinIO starts in |SNSD| mode and creates the zero-parity backend.
 
-.. admonition:: MinIO Avoids Root Disks
-   :class: important
+   * - Existing |SNSD| zero-parity objects and MinIO backend data.
+     - MinIO resumes in |SNSD| mode.
 
-   By default MinIO returns an error if the specified path is part of the root or OS drive.
+   * - Existing filesystem folders, files, and MinIO backend data
```
Comment (djwfyi) [inline source/installation/deploy-minio-single-node-single-drive.rst:45]: All other cells end with a period.

Comment (djwfyi) [reply source/installation/deploy-minio-single-node-single-drive.rst:45]: I think I'd rather the whole table not have periods. The left column does not have any complete sentences.

**Review by djwfyi — APPROVED** [review]
Few minor grumbles, otherwise LGTM
